bemani/backend/iidx/base.py

```diff
@@ -38,9 +38,25 @@ class IIDXBase(CoreHandler, CardManagerHandler, PASELIHandler, Base):
     CHART_TYPE_N14 = 3
     CHART_TYPE_H14 = 4
     CHART_TYPE_A14 = 5
-    # Beginner charts only save status
+    # Beginner charts only save status(not anymore lol)
     CHART_TYPE_B7 = 6
-
+    # HV and above include all the difficulties in one songid now
+    CHART_TYPE_L7 = 7
+    CHART_TYPE_B14 = 8
+    CHART_TYPE_L14 = 9
+    # Used for dealing with mismatch of chart ids from HV and onwards
```

Comment:
What is this doing here? The whole point of having indirection between DB chart and individual game chart mappings is that this is allowed to update, change, get juggled, etc. There shouldn't be a need for mapping updates in the base. See DDR for an example of how songs got completely renumbered, and old SDVX for when they renumbered chart types for some examples.

Reply:
I figured placing it here would make sense since since this is how the songs are going to be from heroic verse onwards. Probably better to put them in each game individually though

Reply:
To clarify, the DB constants should go here in base, and the mapping between constants and their game counterparts (or ignoring charts we don't support for a game) should go in the individual game implementations. It's my fault there's no "GAME_CHART_TYPE_XXXX" for IIDX, they should have been separated like other games. But we have an opportunity to correct it now that you caught me in the act ;)